ci-operator/jobs/openshift/osde2e/openshift-osde2e-master-periodics.yaml

```diff
@@ -1088,6 +1088,53 @@ periodics:
   labels:
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: osde2e-stage-aws-addon-codeready-operator
+  spec:
+    containers:
+    - args:
+      - set -o pipefail; export HOME=$(mktemp -d); export REPORT_DIR=${ARTIFACTS};
+        ./ci/prow_run_tests.sh /usr/local/osde2e-common,/usr/local/crw-osd-token addons;
+      command:
+      - /bin/bash
+      - -c
+      env:
+      - name: ADDON_IDS
+        value: codeready-workspaces
+      - name: ADDON_TEST_HARNESSES
+        value: quay.io/crw/osd-e2e:2.1-latest
+      - name: OSD_ENV
+        value: stage
+      image: registry.svc.ci.openshift.org/openshift/release:golang-1.13
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 10m
+      volumeMounts:
+      - mountPath: /usr/local/osde2e-common
+        name: osde2e-common
+        readOnly: true
+      - mountPath: /usr/local/crw-osd-token
+        name: crw-osd-token
+        readOnly: true
+    serviceAccountName: ci-operator
+    volumes:
+    - name: osde2e-common
+      secret:
+        secretName: osde2e-common
+    - name: crw-osd-token
+      secret:
+        secretName: crw-osd-token
+- agent: kubernetes
+  cluster: api.ci
+  cron: 0 6 * * *
```

Comment:
What timezone is this cron in?
Would be nice to configure it in a way that we have results once we come to work ;-) (and possibly can use the cluster quota)

Reply:
@meowfaceman Can help us with what timezone use the cron?

Reply:
EDT